tests/utils/utils.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * @license Copyright (c) 2003-2015, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+/* globals describe, it, expect, bender */
+
+'use strict';
+
+var modules = bender.amd.require( 'utils' );
+
+describe( 'spy', function() {
+	it( 'should register calls', function() {
+		var utils = modules.utils;
+
+		var fn1 = utils.spy();
+		var fn2 = utils.spy();
+
```

Comment:
I think you miss a check if the `fn1` had the `called` property set to `false` after initialization.